ui/src/lib/ErrorMessage.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+	let {
+		e
+	}: {
+		e: unknown;
+	} = $props();
+</script>
+
+<div class="m-4 text-red-600"><div class="min-w-0">{e}</div></div>
```

Comment:
```suggestion
<div class="m-4 text-destructive"><div class="min-w-0">{e}</div></div>
```

or maybe introduce a error theme dependent error color?